libr/bin/format/xtac/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
No need to ship this file, add a comment in the first line of each source file and you are already using the .license field in the plugin